lib/bolt_server/transport_app.rb

```diff
@@ -630,6 +642,23 @@ def make_winrm_target(target_hash)
       [400, e.message]
     end
 
+    # Returns a list of targets parsed from a Project inventory
+    #
+    # @param project_ref [String] the project_ref to compute the inventory from
+    get '/project_inventory_targets' do
+      bolt_config = config_from_project(params['project_ref'])
+      if bolt_config.inventoryfile && bolt_config.project.inventory_file.to_s != bolt_config.inventoryfile
+        raise Bolt::ValidationError, "Project inventory must be defined in #{bolt_config.project.inventory_file}"
```

Comment:
So this error will give me `<project_ref>/inventory.yaml` file, which could be different from how the user's storing their project because they don't know about `<project_ref>`.

Based on https://puppet.com/docs/bolt/latest/bolt_project_reference.html#inventoryfile, should this error change `#{bolt_config.project.inventory_file}` to `project/inventory.yaml` ?

Reply:
oh yeah, good point.